scripts/check_ipfs_hashes_pushed.py

```diff
@@ -31,11 +31,11 @@
 
 
 IPFS_ENDPOINT = "https://gateway.autonolas.tech/ipfs"
-MAX_WORKERS = 5
-REQUEST_TIMEOUT = 30  # seconds
+MAX_WORKERS = 10
+REQUEST_TIMEOUT = 10  # seconds
```

Comment:
10 workers and 10 seconds should be absolutely fine so reverted it back to the original values
```
2022-12-09T10:51:06.0815534Z OK
2022-12-09T10:51:06.2070945Z check-doc-links-hashes run-test: commands[4] | /home/runner/work/open-autonomy/open-autonomy/scripts/check_ipfs_hashes_pushed.py
2022-12-09T10:51:12.0173742Z Checking hashes for tag 0.5.0.post1
...
2022-12-09T10:51:12.0239933Z Awaiting for results...
2022-12-09T10:51:12.0240630Z The following hashes were not found in IPFS registry: ['bafybeidvxgw3f4hpbmswuifpla4oj2mrqf7je37ar62zqmwuwqvxxzj73m']
2022-12-09T10:51:12.0390225Z ERROR: InvocationError for command /home/runner/work/open-autonomy/open-autonomy/scripts/check_ipfs_hashes_pushed.py (exited with code 1)
2022-12-09T10:51:12.0390693Z ___________________________________ summary ____________________________________
2022-12-09T10:51:12.0391040Z ERROR:   check-doc-links-hashes: commands failed
```
notice it took 6 seconds only to fail